### PR TITLE
Add New Port Plague

### PR DESCRIPTION
**Pull request by MadShmupper**
Add New Port Plague

# New Port for { Plague }

## Game Information
- **Title**: { Plague }
- **URL**: { https://foletto.itch.io/plague }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Review by Cebion — APPROVED** [review]

**Comment by MadShmupper** [comment]
I've just gotten permission to r2r this one so I'll add what's needed this evening

File: ports/plague/cover.png
Comment (JanTrueno) [inline ports/plague/cover.png:1]: Needs to be 640x480.

File: ports/plague/screenshot.png
Comment (JanTrueno) [inline ports/plague/screenshot.png:1]: Also needs to be 640x480.

**Review by JanTrueno — CHANGES_REQUESTED** [review]

File: ports/plague/plague/gamedata/permission.png
Comment (JanTrueno) [inline ports/plague/plague/gamedata/permission.png:1]: Rename this to LICENSE.plague.png to keep some consistency with other licenses. Apart from that seems good :)

**Review by JanTrueno — CHANGES_REQUESTED** [review]

File: ports/plague/plague/gamedata/permission.png
Comment (MadShmupper) [reply ports/plague/plague/gamedata/permission.png:1]: Will do. Ty🎮

**Review by JanTrueno — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**